tools/c7n_org/c7n_org/cli.py

```diff
@@ -610,6 +639,9 @@ def run_account(account, region, policies_config, output_path,
     elif account.get('profile'):
         config['profile'] = account['profile']
 
+    if account.get("oci_compartments"):
+        os.environ[COMPARTMENT_IDS] = account.get("oci_compartments")
```

Comment:
nack, please populate env_vars which setups process environment for the child rather then setting in parent process.

Reply:
This setting of 'COMPARTMENT_IDS' is happening in the run_account() method. If this is not the right place, Can you suggest the location where we need to set this environment variable which is required for the execution.

Reply:
there's an env_vars variable around the same lines that this can be added to which will also clean up the value post policy execution.

Reply:
hmm it seems like that function only gets conditionally called, which seems like its worth revisiting, the logic there looks dated/bitrotted. 

env_vars is also reasonable.

Reply:
Got it...env_vars seems to be the right & safe place. Let me update in that. Thanks.

Reply:
Done